test/test_helper.rb

```diff
@@ -49,19 +49,25 @@ class GhostferryTestCase < Minitest::Test
   def new_ghostferry(filepath, config: {})
     # Transform path to something ruby understands
     path = File.join(GO_CODE_PATH, filepath, "main.go")
-    g = Ghostferry.new(path, config: config, logger: @log_capturer.logger)
+    g = Ghostferry.new(path, config: config, log_capturer: @log_capturer)
+    info("[#{g.tag}] new_ghostferry: create")
     @ghostferry_instances << g
     g
   end
 
   def new_ghostferry_with_interrupt_after_row_copy(filepath, config: {}, after_batches_written: 0)
-    g = new_ghostferry(filepath, config)
+    g = new_ghostferry(filepath, config: config)
 
+    info("[#{g.tag}] new_ghostferry_wiarc: register status hook")
     batches_written = 0
     g.on_status(Ghostferry::Status::AFTER_ROW_COPY) do
       batches_written += 1
+
       if batches_written >= after_batches_written
+        info("[#{g.tag}] new_ghostferry_wiarc: on_status #{batches_written} >= #{after_batches_written} -> true")
         g.send_signal("TERM")
+      else
+        info("[#{g.tag}] new_ghostferry_wiarc: on_status #{batches_written} >= #{after_batches_written} -> false")
```

Comment:
What does _wiarc_ stands for?

Reply:
with interrupt after row copy xD sorry it was just debugging stuff, cleaning it up already